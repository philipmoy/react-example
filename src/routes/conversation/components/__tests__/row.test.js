import React from 'react';
import { shallow } from 'enzyme';

import Row from '../row';

describe('ConversationRow', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <Row
        data={{}}
      />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
