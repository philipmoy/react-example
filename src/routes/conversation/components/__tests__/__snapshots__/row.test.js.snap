// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConversationRow renders correctly 1`] = `
<div
  className="conversation-row"
>
  <div
    className="message"
  >
    <div
      className="avatar-wrapper"
    >
      <div
        className="avatar"
      />
    </div>
    <div
      className="message__content"
    />
  </div>
</div>
`;
